### PR TITLE
Variable declaration added for clarity - only English changes

### PR DESCRIPTION
**Pull request by vrpcse**
Variable declaration is added which is used inside the switch case.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by majestic-owl448 — COMMENTED** [review]
LGTM~! 🥳 Congratulations on making your first pull request!

As a note, our workflow requires at least one additional approval before this can be merged.

**Review by majestic-owl448 — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 
@vrpcse . Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

---

For future contributions, please have a look at the [Contributing Docs](https://contribute.freecodecamp.org/#/index). Particularly the section pertaining to the suggested branch and PR naming conventions. Also, please remember to link the relative issue, if there is one, in the PR description

**Comment by Sky020** [comment]
Closes #40032

**Pull request merged**
